breakglass.go

```diff
@@ -81,6 +82,64 @@ func loadAuthorizedKeys(path string) (map[string]bool, error) {
 	return result, nil
 }
 
+func getEC2MetadataToken() (string, error) {
```

Comment:
Can you move this code into a separate breakglassaws.go please? Also, is there some AWS-provided documentation about these APIs that we could link to in a comment here?

Reply:
Done.